R/reprex_impl.R

```diff
@@ -109,6 +109,20 @@ reprex_impl <- function(x_expr = NULL,
     reprex_file <- rtf_file
   }
 
+  if (venue == "jira") {
+    jira_file <-  files[["jira_file"]]
+    # jira support for pandoc conversion was added in version 2.7.3
+    if (!rmarkdown::pandoc_available("2.7.3")) {
```

Comment:
Let's handle this more like `venue = "rtf"` and highlight. That is:

  * Put the test in a helper `jira_requires_pandoc_2_7_3()`.
  * Check this much earlier.

https://github.com/tidyverse/reprex/blob/b5a049e1660652468715a36d169b379aa1edf29e/R/highlight.R#L16-L25

https://github.com/tidyverse/reprex/blob/b5a049e1660652468715a36d169b379aa1edf29e/R/reprex_impl.R#L20

Reply:
I missed that. This is handle now. I added the help in utils instead of a `jira.R` file with only one function.